bolt-simple-taproot.md

```diff
@@ -0,0 +1,943 @@
+
+Once all partial signatures are known, the `PartialSigAgg` algorithm is used to
+combine the partial signature into a final valid BIP 340 schnorr signature.
+
+### Nothing Up My Sleeves Points
+
+Whenever we want to ensure that a given P2TR can _only_ be spent via a script
+path, we utilize a Nothing Up My Sleeve (NUMs) point. A NUMs point is an EC
+point that no one knows the private key to. If no one knows the private key,
+then it can't be used for key path signing, forcing the script path to always
+be taken.
+
+We refer to the `simple_taproot_nums` as the following value:
+```
+02dca094751109d0bd055d03565874e8276dd53e926b44e3bd1bb6bf4bc130a279
+```
+
+The value was [generated using this tool](https://github.com/lightninglabs/lightning-node-connect/tree/master/mailbox/numsgen), with the seed phrase "Lightning Simple Taproot".
+
+## Design Overview
+
+With the preliminaries out of the way, we provide a brief overview of the
+Simple Taproot Channel design.
+
+The multi-sig output becomes a single P2TR key, with `musig2` key aggregation
+used to combine two keys into one. During the funding flow, public nonces are
+exchanged with the first two messages.
+
+For commitment transactions, we inherit the anchor outputs semantics, meaning
+there're two outputs used for CPFP, with all scripts inheriting a `CSV 1`
+restriction.
+
+The local output of the commitment transaction uses the revocation key as the
+taproot internal key, and commits to a single script path of the delay script.
+
+The remote output of the commitment transaction uses the `simple_taproot_nums`
+as the top-level internal key, and then commits to a normal remote script.
+
+Anchor outputs use the respective local and remote keys from the commitment
+transaction as the top level output, with the committed script being the `CSV
+16` delay.
+
+All HTLC scripts use the revocation key as the top-level internal key. This
+allows the revocation to be executed without additional on-chain bytes, and
+also reduces the amount of state nodes need to keep in order to properly handle
+channel breaches.
+
+## Specification
+
+### Feature Bits
+
+Inheriting the structure put forth in BOLT 9, we define a new feature bit to be
+placed in the `init` message, `node_announcement` and also
+`channel_announcement`. We opt to also place the bit in the
+`channel_announcement` so taproot channels can be identified on the routing
+level, which will eventually matter once PTLCs are fully rolled out.
+
+
+| Bits  | Name                             | Description                                               | Context  | Dependencies      | Link                                  |
+|-------|----------------------------------|-----------------------------------------------------------|----------|-------------------|---------------------------------------|
+| 50/51 | `option_simple_taproot`| Node supports simple taproot chanenls | INC | `option_channel_type`+`option_anchors` | TODO(roasbeef): link |
```

Comment:
Comment from @t-bast: can remove the channel announcement placement here.

Reply:
I don't mind much if we keep it though if you think it has value, it's just that we'll have to define a different feature bit for PTLCs and it's the only one that will matter in `channel_announcement`